src/redist/targets/GenerateRPMs.targets

```diff
@@ -74,15 +73,21 @@
       <SDKTokenValue Include="%SHARED_HOST_RPM_VERSION%">
         <ReplacementString>$(MicrosoftNETCoreAppMajorMinorPatchVersion)</ReplacementString>
       </SDKTokenValue>
-      <SDKTokenValue Include="%ASPNETCORE_SHAREDFX_RPM_PACKAGE_NAME%">
-        <ReplacementString>$(AspNetCoreSharedFxRpmPackageName)</ReplacementString>
-      </SDKTokenValue>
       <SDKTokenValue Include="%ASPNETCORE_SHAREDFX_RPM_PACKAGE_VERSION%">
         <ReplacementString>$(AspNetCoreMajorMinorPatchVersion)</ReplacementString>
       </SDKTokenValue>
       <SDKTokenValue Include="%SHARED_HOST_RPM_NAME%">
         <ReplacementString>$(SharedFxRpmPackageName)</ReplacementString>
       </SDKTokenValue>
+      <SDKTokenValue Include="%SHARED_HOST_TARGETING_PACK_RPM_NAME%">
```

Comment:
Shared host targeting pack? Typo?

Reply:
Copy and pasted the "shared_host" from an existing variable - both are now confusing; resolved.